packages/utils-dev-dependency/cypress/generateDataTests.ts

```diff
@@ -80,7 +80,7 @@ const verifyTransactions = ({
   shouldTestVcEndpoint,
   userId,
 }) => {
-  const url = `/data/aggregator/${aggregator}/user/${userId}/account/${accountId}/transactions${aggregator === "sophtron" ? "?start_time=2021/1/1&end_time=2024/12/31" : ""}`;
+  const url = `/data/aggregator/${aggregator}/user/${userId}/account/${accountId}/transactions${aggregator === "testExampleB" || aggregator === "sophtron" ? "?start_time=2021/1/1&end_time=2024/12/31" : ""}`;
```

Comment:
We need this not care about which aggregator it is. Should we pass in an optional query string parameter?